website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -87,62 +87,58 @@ After making changes, be sure to **restart the docker containers** for the chang
 
 ### Configure Helm installations
 
-You can modify your Appsmith installation on Kubernetes using Helm parameters or Appsmith environment variables. Use the `helm install` command to configure your Appsmith installation using one of the below ways:
+You can modify your Appsmith installation on Kubernetes using a `values.yaml` file. Follow these steps to update the values:
 
-* Using Helm parameters: For example, deploy Appsmith and configure it to use the storage class `appsmith-pv` with:
+<Tabs groupId="appsmithEditions" queryString="current-edition">
+<TabItem label="Community Edition" value="communityEdition"> 
 
+1. Go to the root directory of your installation.
 
-```bash
-helm upgrade \
-#highlight-next-line
---set persistence.storageClass=appsmith-pv \
-  appsmith appsmith/appsmith 
-```
+2. Generate `values.yaml` file with:
 
-* **Or** using Appsmith environmental variables: For example, change the encryption salt configuration with:
+  ```bash
+  helm show values appsmith/appsmith > values.yaml
+  ```  
 
-```bash 
-helm upgrade \
-#highlight-next-line
---set applicationConfig.APPSMITH_MAIL_FROM="test@test.com" \
-  appsmith appsmith/appsmith 
-```
+3. Modify the parameter values in the `values.yaml` file available under the `applicationConfig` section. For example, if you want to change the email address from which your messages are sent, as shown below:
 
-* **Or** using `values.yaml` file: 
+  ```yaml
+  applicationConfig:
+    #highlight-next-line
+    APPSMITH_MAIL_FROM:"test@test.com"
+  ```
 
-<Tabs groupId="appsmithEditions" queryString="current-edition">
-<TabItem label="Community Edition" value="communityEdition"> 
+4. Update the values with:
 
-1. Update the parameter values in the `values.yaml` file under the `applicationConfig` section:
+  ```bash
+  helm upgrade appsmith appsmith/appsmith -f values.yaml 
```

Comment:
helm upgrade appsmith appsmith/appsmith -f values.yaml -n appsmith

Reply:
These changes are done as part of #1295. These will get merged to main with that PR.